samples/shopping-cart-addview/.gitignore

```diff
@@ -0,0 +1,26 @@
+# Eclipse
+.classpath
+.project
+.settings/
+
+# Intellij
+.idea
+!.idea/
+.idea/*
+!.idea/runConfigurations/Run_locally.xml
```

Comment:
also include this
```
.idea/runConfigurations/Run_locally.xml
```

if it's still ignored by .gitignore you might have to force add it